src/components/Layout/Header.tsx

```diff
@@ -3,40 +3,62 @@
 import Link from 'next/link';
 
 import { useAuth } from '../../context/AuthContext';
+import { LogoIcon } from '../ui/icons/LogoIcon';
+import { ProfileIcon } from '../ui/icons/ProfileIcon';
+import { ArrowIcon } from '../ui/icons/ArrowIcon';
 
 export const Header: React.FC = () => {
   const { user, logout } = useAuth();
-  const { name, email } = user;
+  const { id, name, email } = user;
 
   return (
-    <header className="fixed w-full bg-white">
-      <nav className="flex flex-row items-center justify-between">
-        <Link href="/" className="px-4 py-8 text-3xl md:px-8">
-          SportujSpolu
+    <header className="fixed top-0 z-header w-full bg-white">
+      <nav className="flex flex-row items-center justify-between border border-light-gray bg-lightest-gray">
+        <Link href="/" className="px-4 py-4 text-3xl md:px-14">
+          <LogoIcon className="hover:fill-primary" />
         </Link>
```

Comment:
alt

Reply:
I think that would work only for <img> not <svg> (?) 👀